Python/Product/Cookiecutter/View/CookiecutterOptionsPage.xaml.cs

```diff
@@ -80,6 +80,17 @@ internal partial class CookiecutterOptionsPage : Page {
 
             ViewModel.CreateFilesAsync().DoNotWait();
         }
+
+        private void OpenInBrowser_CanExecute(object sender, CanExecuteRoutedEventArgs e) {
+            var url = (string)e.Parameter;
+            e.CanExecute = !string.IsNullOrEmpty(url);
+            e.Handled = true;
+        }
+
+        private void OpenInBrowser_Executed(object sender, ExecutedRoutedEventArgs e) {
+            var url = (string)e.Parameter;
+            Process.Start(url)?.Dispose();
```

Comment:
This is a security risk, given it's an arbitrary string from an untrusted source. We should at least `Uri.Parse` it and make sure the scheme is http or https, and probably use of the internal browser rather than passing the string to the shell.